tests/lib/test-crio.bash.in

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# CC tests data
+TEST_DATA="${BATS_TEST_DIRNAME}/../../data/"
+
+# CRI-O URL and version
+CRIO_GITHUB="github.com/kubernetes-incubator/cri-o"
+CRIO_VERSION="3195f459"
+
+# CRI-O root directory
+OCID_ROOT="$GOPATH/src/$CRIO_GITHUB"
+
+# Path of the ocid binary.
+OCID_BINARY=${OCID_BINARY:-${OCID_ROOT}/ocid}
+# Path of the ocic binary.
+OCIC_BINARY=${OCIC_BINARY:-${OCID_ROOT}/ocic}
+# Path of the conmon binary.
+CONMON_BINARY=${CONMON_BINARY:-${OCID_ROOT}/conmon/conmon}
```

Comment:
Wow, I'm kinda surprised they have a binary with that name - sooo easy to mis-type as `common`! :smile: